src/pages/patientView/clinicalInformation/PatientViewPageStore.ts

```diff
@@ -1310,7 +1310,11 @@ export class PatientViewPageStore {
         {
             await: () => [this.molecularProfilesInStudy],
             invoke: async () =>
-                findMrnaRankMolecularProfileId(this.molecularProfilesInStudy),
+                findMrnaRankMolecularProfileId(
```

Comment:
i just changed this function to accept a list of ids instead of whole promise, which was silly and complicated testing